addons/mail/views/mail_template_views.xml

```diff
@@ -15,82 +15,47 @@
                     </header>
                     <sheet>
                         <div class="oe_button_box" name="button_box">
-                            <field name="ref_ir_act_window" invisible="1"/>
-                            <button class="oe_stat_button"
-                                    groups="base.group_system"
-                                    name="create_action" type="object"
-                                    attrs="{'invisible':[('ref_ir_act_window','!=',False)]}" icon="fa-plus"
-                                    help="Display an option on related documents to open a composition wizard with this template">
-                                <div class="o_field_widget o_stat_info">
-                                    <span class="o_stat_text">Add</span>
-                                    <span class="o_stat_text">Context Action</span>
-                                </div>
-                            </button>
-                            <button name="unlink_action" type="object"
-                                    groups="base.group_system"
-                                    class="oe_stat_button" icon="fa-minus"
-                                    attrs="{'invisible':[('ref_ir_act_window','=',False)]}"
-                                    help="Remove the contextual action to use this template on related documents" widget="statinfo">
-                                <div class="o_field_widget o_stat_info">
-                                    <span class="o_stat_text">Remove</span>
-                                    <span class="o_stat_text">Context Action</span>
-                                </div>
-                            </button>
                             <button class="oe_stat_button" name="%(mail_template_preview_action)d" icon="fa-search-plus" string="Preview"
                                     type="action" target="new"/>
                         </div>
                         <div class="oe_title">
                             <label for="name"/>
-                            <h1><field name="name" required="1" placeholder='e.g. "Welcome email"'/></h1>
+                            <h1><field name="name" class="w-100"
+                                required="1" placeholder='e.g. "Welcome email"'/></h1>
                             <group>
                                 <field name="model_id" placeholder="e.g. Contact" required="1" options="{'no_create': True}"/>
+                                <field name="subject" placeholder='e.g. "Welcome to MyCompany" or "Nice to meet you, ${object.name}"'/>
```

Comment:
We could upate the placeholder, as this still contains jinja that is deprecated since Odoo V15 (should be ``{{ object.name }}`` ).